tests/unit/search/test_filters.py

```
@@ -1,12 +1,65 @@
+from fastapi import FastAPI
 from pydantic.error_wrappers import ValidationError
```

Pylint (4.0.6)

[error] No name 'ValidationError' in module 'pydantic.error_wrappers'

(E0611)